### PR TITLE
chore: added temporary static files and a bit of text for clarity

### PR DESCRIPTION
**Pull request by ViorelMocanu**
Initial commit for future meta improvements.

### CODE REVIEW

File: src/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg width="1080" height="1080" viewBox="0 0 1080 1080" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (CosticaPuntaru) [inline src/favicon.svg]: isn't there a limitation and favicon must be .ico file? at least for some browsers?

File: src/js/main.js [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
 import image from '../assets/images/image.jpg';
 
 console.log('main js file works');
-console.log(' here is the path for a asset in js', image)
-printImportWorks()
+console.log('here is the path for a asset in js', image);
+printImportWorks();
+
+if ('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('../service-worker.js').then(function (registration) {
```
Comment (CosticaPuntaru) [inline src/js/main.js:11]: as this aims to be for beginers and only for playgrounds,  not sure if service workers should be added

File: src/index.html [hunk 3]
```diff
@@ -1,43 +1,174 @@
...
+	<meta name="og:image:height" content="630" />
+	<meta name="og:email" content="$site_email" />
+	<meta name="og:phone_number" content="$site_telephone" />
+	<meta name="article:publisher" content="$site_name" />
+	<meta name="fb:app_id" content="$facebook_appid" >
+	<meta name="twitter:site" content="$twitter_site">
+	<meta name="twitter:creator" content="$twitter_account">
+	<meta name="twitter:card" content="summary_large_image" />
+	<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+	<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
```
Comment (CosticaPuntaru) [inline src/index.html:30]: if this is intended to be used for portofolios or smll js demobs, not sure we should include this many branding images, the user will need to change/remove all of them, simplicity should be a focus of the project

File: src/styles/reset.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/* The SCSS or CSS you write in this file will be merged with all the other files you reference when deploying */
+/* Including this second file you imported via main.scss */
+
```
Comment (CosticaPuntaru) [inline src/styles/reset.scss:3]: please add a proper scss reset :)

File: src/favicon.svg (hunk 1 above)
Comment (ViorelMocanu) [reply src/favicon.svg]: Yep. https://caniuse.com/link-icon-svg This is why there are several `<link rel="icon" ... />` instances (line 32 is a fallback for line 33).

File: src/js/main.js (hunk 2 above)
Comment (ViorelMocanu) [reply src/js/main.js:11]: OK, but this only adds the static files in a local cache, it's a good thing to have. We can remove it if it doesn't make sense, but it helped me rank better in Lighthouse for several previous projects...

File: src/index.html (hunk 3 above)
Comment (ViorelMocanu) [reply src/index.html:30]: Unfortunately, when it comes to standardizing the meta images and icons, this is necessary to cover all the browsers and versions. I will link to https://realfavicongenerator.net/ so it's as easy as possible to generate everything based on a single image.

File: src/styles/reset.scss (hunk 4 above)
Comment (ViorelMocanu) [reply src/styles/reset.scss:3]: Yep. On it. :)

**Pull request merged**